packages/server/package.json

```diff
@@ -131,9 +131,9 @@
     "relay-runtime": "^11.0.2",
     "rethinkdb-ts": "2.4.5",
     "rethinkdbdash": "^2.3.31",
-    "samlify": "^2.6.1",
+    "samlify": "2.7.7",
     "secure-compare": "^3.0.1",
-    "sharp": "^0.26.2",
+    "sharp": "^0.28.0",
```

Comment:
Needed for the m1 chip. I'll create a separate PR with these changes once this is merged: https://github.com/ParabolInc/parabol/pull/5396